cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -195,7 +197,8 @@ func TestFileExistingCheck(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to create file: %v", err)
 	}
-	defer os.Remove(f.Name())
+	defer utiltesting.CloseAndRemove(t, f)
```

Comment:
This function is not defined in this commit?  Ideally we build and pass tests at each commit

Reply:
> This function is not defined in this commit? Ideally we build and pass tests at each commit

This function is in this commit, it's in `staging/src/k8s.io/client-go/util/testing/remove_file.go` and this commit has passed all the tests

Reply:
uhh, we must be looking at different things - the github UI shows https://github.com/kubernetes/kubernetes/pull/116656/commits/aa120418dbbcd6d239d1ed082355ee18d26a15b3 as the first commit, and this does not include this function

Reply:
This function seems to be defined in the very last commit

Reply:
This is still weird.  The first commit to use this function should define it, or else the tree doesn't build at that commit.

Reply:
> This is still weird. The first commit to use this function should define it, or else the tree doesn't build at that commit.

I understand and have corrected it